src/System.Private.Xml/src/System/Xml/Core/XmlTextWriter.cs

```diff
@@ -1367,9 +1403,12 @@ private void WriteEndStartTag(bool empty)
             _xmlEncoder.EndAttribute();
             if (empty)
             {
-                _textWriter.Write(" /");
+                _textWriter.Write(s_closeTagEnd);
```

Comment:
StreamWriter.Write(char[]) is faster than StreamWriter.Write(string)

---
In reply to: [122563827](https://github.com/dotnet/corefx/pull/21155#discussion_r122563827) [](ancestors = 122563827)